### PR TITLE
File readme.MD is updated with Carla Info

### PR DESCRIPTION
**Pull request by cclics**
## What type of PR is this? (check all applicable)

- [ ] Feature
- [ ] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ x] Documentation Update

## Description of what PR does
The file README.md (id: 463f7cf) with Carla's information.

## Related PRs (if any)
-
-

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed _(please include reasons for why tests aren't needed)_
- [ ] no, because I need help

## Added to documentation?

- [ ] Yes, project README
- [ ] No documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Alt-Text

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -20,6 +20,8 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  user: corcovado
```
Comment (msarit) [inline config/database.yml:23]: @drippdropp Hey Daniel, I wouldn't think that we should be committing additions like this to the general codebase? 🤔  How do we avoid this?

**Comment by msarit** [comment]
@cclics Hey Carla, please resolve your conflicts so I can merge this PR 🙏🏾

**Comment by cclics** [comment]
Hi Arit @msarit I resolved the conflict. I will create it in a  branch for with changes

**Comment by cclics** [comment]
Hi Arit @msarit, I created a new PR issue #35 (branch from cclics-update-readme) to substitute this PR.

**Comment by msarit** [comment]
Closing this PR per Carla's comment [here](https://github.com/OurTimeForTech/shiftwork/pull/32#issuecomment-802312686)

**Pull request closed without merging**